app/controllers/admin/employees_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Admin
+  class EmployeesController < AdminController
+    before_action :find_employee, only: %i[show edit update destroy]
```

Comment:
actually using callbacks is making code harder to read - treat it as FYI, let's leave it like this for now